### PR TITLE
improve chart, refactory and minor fix

### PR DESCRIPTION
**Pull request by DavideSegullo**
## What is the purpose of the change:

- Refactor hook token detail page
- Removed the use of client-side assetlists and chainlists in token detail pages
- Reduce prop drilling by using prefetch
- Improved directory management 
- Added new header for the graph
- Added new graph style
- Added new footer with chart filters

### Linear Task

[Linear Task URL](https://linear.app/osmosis/issue/FE-504/add-volume-charts-to-asset-info-page)
[Linear Task URL](https://linear.app/osmosis/issue/FE-537/add-chart-type-select)
[Linear Task URL](https://linear.app/osmosis/issue/FE-556/add-redirect-to-assets-page-on-404-error)

### CODE REVIEW

File: packages/web/components/chart/price-historical-v2.tsx [hunk 1]
```diff
@@ -58,3 +64,41 @@ export const HistoricalPriceChartV2: FunctionComponent<{
     />
   );
 });
+
+export const HistoricalPriceChartHeaderV2: FunctionComponent<{
```
Comment (jonator) [inline packages/web/components/chart/price-historical-v2.tsx:68]: Would be good to eventually replace v1 with v2 for cleanup

File: packages/web/components/pages/asset-info-page/token-details.tsx [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
-  const language = useCurrentLanguage();
-  const { queriesExternalStore, priceStore, chainStore } = useStore();
+  const { queriesExternalStore, priceStore } = useStore();
   const { logEvent } = useAmplitudeAnalytics();
 
-  const details = useMemo(() => {
-    return tokenDetailsByLanguage
-      ? tokenDetailsByLanguage[language]
-      : undefined;
-  }, [language, tokenDetailsByLanguage]);
```
Comment (jonator) [inline packages/web/components/pages/asset-info-page/token-details.tsx:45]: Nice. I think there were a bit too many useMemo hooks in use

File: packages/web/components/pages/asset-info-page/your-balance.tsx
Comment (jonator) [inline packages/web/components/pages/asset-info-page/your-balance.tsx:1]: Nice, was wanting to move this into a folder

File: packages/web/components/chart/price-historical-v2.tsx (hunk 1 above)
Comment (DavideSegullo) [reply packages/web/components/chart/price-historical-v2.tsx:68]: Ye I'll do it, right now the old header is still used for the CL pools, so maybe I'll do it after the info page PR

**Review by jonator — APPROVED** [review]

**Review by CryptoAssassin1 — APPROVED** [review]

**Pull request merged**